package.json

```diff
@@ -3,14 +3,21 @@
   "version": "2.3.0",
   "main": "lib/index.js",
   "scripts": {
+    "build:test": "docker-compose build test",
+    "build:web": "docker-compose build web",
+    "docker:lint": "docker-compose run test npm run lint",
+    "docker:start": "docker-compose up -d web",
+    "docker:test": "docker-compose run test npm test",
+    "docker:test:ci": "docker-compose up --build test_base",
     "start": "node server",
     "test": "lab -a @hapi/code -L",
     "lint": "eslint ."
   },
   "dependencies": {
     "@hapi/boom": "7.x.x",
     "@hapi/joi": "15.x.x",
-    "haute-couture": "3.x.x"
+    "haute-couture": "3.x.x",
+    "nodemon": "1.x.x"
```

Comment:
Could/should nodemon be a dev dependency?

Reply:
https://hapihour.slack.com/archives/CA4J8PBGR/p1565830920088700?thread_ts=1565804620.085300&cid=CA4J8PBGR

Conversation in thread above. 

tl;dr: yes, but it would make the build process more complicated and differ between local/stg/prod, and I generally suggest people try not to use different processes across environments if possible because it reduces the likelihood of error.